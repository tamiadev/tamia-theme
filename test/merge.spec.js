const merge = require('../merge');

test('merge objects', () => {
	const result = merge({ a: 1, b: 2 }, { c: 3 }, { a: 42 });
	expect(result).toMatchSnapshot();
});
