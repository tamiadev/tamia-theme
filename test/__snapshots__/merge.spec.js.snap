// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`merge objects 1`] = `
Object {
  "a": 42,
  "b": 2,
  "c": 3,
}
`;
